Sources/AmazonLocationiOSAuthSDK/Types/ApiTypes/ResponseTypes/EmptyResponse.swift

```diff
@@ -0,0 +1,4 @@
+public struct EmptyResponse: Codable {
```

Comment:
Doesnt seem so empty if theres description

Reply:
Changed the name